persist/sqlite/init.sql

```diff
@@ -327,6 +328,37 @@ CREATE TABLE v2_transaction_file_contract_revisions (
 );
 CREATE INDEX v2_transaction_file_contract_revisions_transaction_id_index ON v2_transaction_file_contract_revisions(transaction_id);
 
+CREATE TABLE v2_transaction_file_contract_resolutions (
+    transaction_id INTEGER REFERENCES v2_transactions(id) ON DELETE CASCADE NOT NULL,
+    transaction_order INTEGER NOT NULL,
+    parent_contract_id INTEGER REFERENCES v2_file_contract_elements(id) ON DELETE CASCADE NOT NULL, -- add an index to all foreign keys
```

Comment:
Do we have sufficient indices on this table?

Reply:
I don't think so because we only query this table by transaction_id.